.ci.yaml

```diff
@@ -2609,6 +2609,20 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Mac flavors_test_macos
```

Comment:
I tried to run this locally and it got stuck on:
```
[flavors_test_macos] [STDOUT] stdout: 02:22 +0: loading /Users/m/Projects/flutter/dev/integration_tests/flavors/integration_test/integration_test.dart
[flavors_test_macos] [STDOUT] stdout: 02:23 +0: loading /Users/m/Projects/flutter/dev/integration_tests/flavors/integration_test/integration_test.dart
[flavors_test_macos] [STDOUT] stdout: 02:24 +0: loading /Users/m/Projects/flutter/dev/integration_tests/flavors/integration_test/integration_test.dart
[flavors_test_macos] [STDOUT] stdout: 02:25 +0: loading /Users/m/Projects/flutter/dev/integration_tests/flavors/integration_test/integration_test.dart
[flavors_test_macos] [STDOUT] stdout: 02:26 +0: loading /Users/m/Projects/flutter/dev/integration_tests/flavors/integration_test/integration_test.dart
```

Did you see this locally when you ran it?  I'm not sure why it wouldn't be loading in my environment...

Reply:
Running `dart ./bin/run.dart -t flavors_test_macos` hangs on 

```
[flavors_test_macos] [STDOUT] The Dart VM service is listening on http://127.0.0.1:60212/-JV7h6ggCrk=/
[flavors_test_macos] Connected to VM server.
[flavors_test_macos] [STDOUT] Running task with a timeout of null.
[flavors_test_macos] [STDOUT] 
[flavors_test_macos] [STDOUT] 
[flavors_test_macos] [STDOUT] ══════════════════╡ ••• Checking running Dart processes ••• ╞═══════════════════
[flavors_test_macos] [STDOUT]
```

I don't think that I'm getting to the same point that you are...